wandb/apis/public.py

```diff
@@ -295,6 +310,69 @@ class User(Attrs):
     def init(self, attrs):
         super(User, self).__init__(attrs)
 
+class Projects(Paginator):
+    QUERY = gql('''
+        query Projects($entity: String, $cursor: String, $perPage: Int = 50) {
+            models(entityName: $entity, after: $cursor, first: $perPage) {
+                edges {
+                    node {
+                        ...ProjectFragment
+                    }
+                    cursor
+                }
+                pageInfo {
+                    endCursor
+                    hasNextPage
+                }
+            }
+        }
+        %s
+        ''' % PROJECT_FRAGMENT)
+
+    def __init__(self, client, entity, per_page=50):
+        self.entity = entity
+        variables = {
+            'entity': self.entity,
+        }
+        super(Projects, self).__init__(client, variables, per_page)
+
+    @property
+    def length(self):
+        # TODO: We don't have a project count in graphql right now
+        return 100000
+        return None
```

Comment:
return None?

Reply:
Yeah I want to fix the TODOs in the top of this PR before merging.